tests/test_guitarset.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+import os
+import pytest
+
+from mirdata import guitarset, utils
+from tests.test_utils import DEFAULT_DATA_HOME
+
+TEST_DATA_HOME = 'tests/resources/mir_datasets/GuitarSet'
+TRACK = guitarset.Track('03_BN3-119-G_solo', data_home=TEST_DATA_HOME)
+
+
+def test_track_basic():
+    # test data home None
+    track_default = guitarset.Track('03_BN3-119-G_solo')
+    assert track_default._data_home == os.path.join(DEFAULT_DATA_HOME, 'GuitarSet')
+
+    with pytest.raises(ValueError):
+        guitarset.Track('asdfasdf', data_home=TEST_DATA_HOME)
+    
+    # test __repr__
+    assert isinstance(track_default.__repr__(), str)
+
+
+def test_track_simple_attributes():
+    # test attributes
+    assert TRACK.track_id == '03_BN3-119-G_solo'
+    assert TRACK._data_home == TEST_DATA_HOME
+    assert os.path.isfile(TRACK.audio_hex_cln_path)
+    assert os.path.isfile(TRACK.audio_hex_path)
+    assert os.path.isfile(TRACK.audio_mic_path)
+    assert os.path.isfile(TRACK.audio_mix_path)
+    assert os.path.isfile(TRACK.jams_path)
+    assert TRACK.player_id == '03'
+    assert TRACK.tempo == 119
+    assert TRACK.mode == 'solo'
+    assert TRACK.style == 'Bossa Nova'
+    
+
```

Comment:
These tests are mostly testing that the types are correct - can you add some small tests that specifically call the different loading functions, e.g. `_load_beats()` and make sure everything behaves as expected?

Reply:
You caught me being lazy hahaha :)